source/plugins/go/src/telemetry.go

```diff
@@ -254,7 +254,12 @@ func SendContainerLogPluginMetrics(telemetryPushIntervalProperty string) {
 				TelemetryClient.Track(logLatencyMetric)
 			}
 		}
-		TelemetryClient.Track(appinsights.NewMetricTelemetry(metricNameNumberofTelegrafMetricsSentSuccessfully, telegrafMetricsSentCount))
+		telegrafEnabled := make(map[string]string)
```

Comment:
the reason why we use a map instead of boolean is because we might expand this later?